src/gatling/simulations/scenarios/ET_MakeAClaimPt2.scala

```diff
@@ -210,8 +210,8 @@ object ET_MakeAClaimPt2 {
         .post(BaseURL + "/pay")
         .headers(CommonHeader)
         .header("content-type", "application/x-www-form-urlencoded")
-        .formParam("_csrf", "${csrf}")
-        .formParam("et-sya-session", "${etSession}")
+        .formParam("_csrf", "#{csrf}")
+        .formParam("et-sya-session", "#{etSession}")
         .formParam("payBeforeTax", now.plusYears(5 + rnd.nextInt(15)).format(patternYear))
```

Comment:
Because these random numbers are generated at compile time, I think they'll be the same for every user. If you want them to be random, you'll need to set them into a gatling session variable at the top of the script with the others.

Reply:
Makes sense, I have changed it so it is generated at the start of the session